pkg/inspector/image-inspector.go

```diff
@@ -103,25 +104,33 @@ func NewDefaultImageInspector(opts iicmd.ImageInspectorOptions) ImageInspector {
 
 // Inspect inspects and serves the image based on the ImageInspectorOptions.
 func (i *defaultImageInspector) Inspect() error {
-	client, err := docker.NewClient(i.opts.URI)
-	if err != nil {
-		return fmt.Errorf("Unable to connect to docker daemon: %v\n", err)
-	}
+	if i.opts.UseDockerSocket {
+		client, err := docker.NewClient(i.opts.DockerSocket)
+		if err != nil {
+			return fmt.Errorf("Unable to connect to docker daemon: %v\n", err)
```

Comment:
nit: in origin we have convention that fmt.Errorf is always lowecase as you then prefix them with 'ERROR:' or something at higher level.